middlewares/validators/tasks.js

```diff
@@ -133,8 +134,28 @@ const updateSelfTask = async (req, res, next) => {
   }
 };
 
+const getTasks = async (req, res, next) => {
+  const schema = joi.object().keys({
+    dev: joi.bool().optional().sensitive(),
+    status: joi
+      .string()
+      .insensitive()
+      .valid(...MAPPED_TASK_STATUS_ENUM)
+      .optional(),
+  });
+
+  try {
+    await schema.validateAsync(req.query);
+    next();
+  } catch (error) {
+    logger.error(`Error validating getTasks query : ${error}`);
+    res.boom.badRequest(error.details[0].message);
```

Comment:
@RitikJaiswal75 send me once...sharing here also
<img width="604" alt="image" src="https://github.com/Real-Dev-Squad/website-backend/assets/107163260/45d16cfa-9981-4458-9cf7-afec6f5237bd">

Reply:
The problem is that in this case, the error could be because of invalid query param, or invalid values for an existing query params. Creating dynamic messages for every case might not be scalable: as the query params increases, there can be too many cases to handle.